Makefile

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Silicon Laboratories Inc.
```

Comment:
Then I don't see the point in having it as a makefile, nor for having the capability of running sims from it.

If we are calling it like a script, it might as well be a script.

If we have to cd to use it, then one might as well cd to the directory we are already cd'ing to.

@silabs-hfegran what do you think about moving it to mk/?

Reply:
> If we are calling it like a script, it might as well be a script.

I am OK with that.

Reply:
This is somewhat different from the initial idea we had, where the top level makefile could just seemlessly call the required targets.

What I would like to avoid though, is a separate manual step to get the appropriate code in place, especially if that command has to be run from a different place than the normal make commands (`./<cv_core>/sim/uvmt.`) Unfortunately that directory does not exist prior with the new branches, thus we need a way to get this in place prior to running the other commands.

I am not opposed to placing files in a different place, but I believe there needs to be a logical place (for make this is normally top level) to run the required command.